src/docs/reference/modules/Email.Azure/README.md

```diff
@@ -1,26 +1,27 @@
 # Azure Email (`OrchardCore.Email.Azure`)
 
-This module provides an Email provider for sending emails through the [Azure Communication Services Email](https://learn.microsoft.com/en-us/azure/communication-services/concepts/email/email-overview).
+This module provides an Email provider for sending emails through the [Azure Communication Email Services](https://learn.microsoft.com/en-us/azure/communication-services/concepts/email/email-overview).
```

Comment:
The original was good, see https://learn.microsoft.com/en-us/azure/communication-services/concepts/email/email-overview#key-principles-of-azure-communication-services-email.

```suggestion
This module provides an Email provider for sending emails through [Azure Communication Services Email](https://learn.microsoft.com/en-us/azure/communication-services/concepts/email/email-overview).
```